re/mocks/auth.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Mainflux
```

Comment:
If you picked this header, which is nice and simple - please align all others to this